docker/oso-host-monitoring/src/root/rkhunter.yml.j2

```diff
@@ -0,0 +1,97 @@
+{{ generated_header }}
```

Comment:
If we wanted to be real consistent, we could call this file rkhunter_config.yml to match the usual config.yml playbook, but it's fine as-is.

Reply:
Very much agree to this.  Name this rkhunter_config.yml, and rename the other one to FOO_config.yml (for an appropriate value of foo).